projects/treasury/jpegd.js

```diff
@@ -25,17 +25,18 @@ const treasuryTvl = treasuryExports({
       ADDRESSES.ethereum.LINK,
       ADDRESSES.ethereum.cvxCRV,
       ADDRESSES.ethereum.SAFE,
-      "0xC47EBd6c0f68fD5963005D28D0ba533750E5C11B",
-      "0x34eD182D0812D119c92907852D2B429f095A9b07",
-      "0x836A808d4828586A69364065A1e064609F5078c7",
-      "0x9848482da3Ee3076165ce6497eDA906E66bB85C5",
       ADDRESSES.ethereum.vlCVX,
       ADDRESSES.ethereum.sfrxETH,
       ADDRESSES.ethereum.STETH,
       ADDRESSES.ethereum.WSTETH,
       ADDRESSES.ethereum.RETH,
-      lp,
-      lp2,
+      "0x836A808d4828586A69364065A1e064609F5078c7", // pETH
+      // Liquidity positions
+      "0xC47EBd6c0f68fD5963005D28D0ba533750E5C11B", // pUSDFRAXBP3CRV-f
+      "0x34eD182D0812D119c92907852D2B429f095A9b07", // JPEGETH-f
+      "0x9848482da3Ee3076165ce6497eDA906E66bB85C5", // pETH-ETH-f
+      "0xdB06a76733528761Eda47d356647297bC35a98BD", // JPEGWETH SLP
+      "0xC36442b4a4522E871399CD717aBDD847Ab11FE88", // UniV3 positions
```

Comment:
you can skip this as it is already included thanks to this flag `resolveUniV3: true,`

Reply:
removed